docs/topics/proc_git-workflow-for-making-changes.adoc

```diff
@@ -29,12 +29,7 @@ If there are errors in validation, open the `master.xml` file in the directory o
 
 If you are unsure, ask someone from the team for help.
 --
-. Create a pull request against the branch you based your contribution off. Ensure you also:
-+
---
-.. Label the pull request appropriately. For more information about labels, see xref:using-github-labels_{context}[].
-.. Reference the issue that your pull request resolves in the description, using the formulation `resolves $ID`, where `$ID` is the numerical ID of the issue. Doing this automatically closes the issue when the pull request is accepted.
---
+. Create a pull request against the branch you based your contribution off.Reference the issue that your pull request resolves in the description, using the formulation `resolves $ID`, where `$ID` is the numerical ID of the issue. Doing this automatically closes the issue when the pull request is accepted.
```

Comment:
Well done:) I think  it might be safe for you to remove a the section about referencing issues IDs on PRs. It no longer applies to our project because we switched to tracking issues in JIRA.